catalog/monitors/aws-service-check.yaml

```diff
@@ -0,0 +1,31 @@
+# The official Datadog API documentation with available query parameters & alert types:
```

Comment:
maybe rename the file to `aws.yaml` under https://github.com/cloudposse/terraform-datadog-monitor/blob/master/catalog/monitors/